core/src/main/scala/org/apache/spark/util/SignalUtils.scala

```diff
@@ -60,10 +60,11 @@ private[spark] object SignalUtils extends Logging {
     if (SystemUtils.IS_OS_UNIX) {
       try {
         val handler = handlers.getOrElseUpdate(signal, {
-          logInfo("Registered signal handler for " + signal)
+          logInfo("Registering signal handler for " + signal)
           new ActionHandler(new Signal(signal))
         })
         handler.register(action)
+        logInfo("Registered signal handler for " + signal)
```

Comment:
This seems unnecessary. If registration fails you'll get an error message from the exception handler below, no? Then the previous message is enough for the success case.

Reply:
So strangely enough not always while I was working on this (before I'd settled on SIGPWR) -- but that's likely no longer important corner case, I'll remove this.